tortoise/functions.py

```diff
@@ -103,6 +103,13 @@ def _resolve_field_for_model(self, model: "Type[Model]", table: Table, field: st
 
         return {"joins": joins, "field": field}
 
+    def _resolve_default_values(self, model: "Type[Model]", table: Table) -> Iterator[Any]:
```

Comment:
What's for this?

Reply:
Iterate over default values if it meet a function.  For concat operation , it may happen to exist a function in positon argument later than first argument